cmd/kubelet/app/options/options.go

```diff
@@ -417,6 +417,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 		"Minimum TLS version supported. "+
 			"Possible values: "+strings.Join(tlsPossibleVersions, ", "))
 	fs.BoolVar(&c.RotateCertificates, "rotate-certificates", c.RotateCertificates, "Auto rotate the kubelet client certificates by requesting new certificates from the kube-apiserver when the certificate expiration approaches.")
+	fs.BoolVar(&c.MergeDefaultEvictionSettings, "merge-default-eviction-settings", c.MergeDefaultEvictionSettings, "It will inherit the default values for the unchanged parameters of eviction hard in case any of the parameter is changed")
```

Comment:
for new kubelet knobs, we typically just add the config field field now, new flags are discouraged

Reply:
I have removed this from here